scripts/pre-commit.sh

```diff
@@ -1,15 +1,7 @@
 #!/bin/sh
 
-# find staged files
-stagedFiles=$(git diff --staged --name-only)
+R='\033[0;31m'
```

Comment:
This looks black magic to me. :-)

Reply:
It's just red color so it's a bit more pretty

Reply:
It looks like I forgot to approve.